src/init/register_configure_agent.sh

```diff
@@ -9,10 +9,12 @@
 # Foundation.
 
 . /etc/ossec-init.conf
-  
-edit_value_tag() { 
+
+sed="sed -ri"
+
+edit_value_tag() {
     if [ "$#" == "2" ] && [ ! -z "$2" ]; then
-        sed -ri "s#<$1>.+</$1>#<$1>$2</$1>#g" "${DIRECTORY}/etc/ossec.conf" > /dev/null
+        ${sed} "s#<$1>.*</$1>#<$1>$2</$1>#g" "${DIRECTORY}/etc/ossec.conf"
```

Comment:
Are we sure that `> /dev/null` is not necessary here?

Reply:
Hi @alberpilot, `sed -i` doesn't return any output, so redirecting the output to `/dev/null` isn't necessary. In addition, in MacOS X hosts, installing a package doesn't log any output in the stdout. Every output (stdout and stderr) are logged in `/var/log/install.log`. 

I can add it again if you want :smiley: